### PR TITLE
Bump pulumi/pkg and pulumi/sdk to 3.143.0

### PR DESCRIPTION
**Pull request by rshade**

### CODE REVIEW

**Comment by justinvp** [comment]
We typically update the `pulumi` git submodule at the same time as bumping these. Want to do that?

**Comment by rshade** [comment]
@justinvp yes sir, `should` be updated to the 3.143.0 tag.

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ### Improvements
 
+- Update to Pulumi 3.139.0
```
Comment (justinvp) [inline CHANGELOG_PENDING.md:3]:
```suggestion
- Update to Pulumi 3.143.0
```

**Review by justinvp — APPROVED** [review]
LGTM otherwise

**Pull request merged**